src/TypesFinder/PhpDocumentor/NamespaceNodeToReflectionTypeContext.php

```diff
@@ -35,17 +35,28 @@ public function __invoke(?Namespace_ $namespace) : Context
     private function aliasesToFullyQualifiedNames(Namespace_ $namespace) : array
     {
         // flatten(flatten(map(stuff)))
-        return array_merge([], ...array_merge([], ...array_map(function ($use) : array {
-            /** @var $use Use_|GroupUse */
+        return array_merge(
```

Comment:
I don't understand what was changed here, since the alignment was changed. The operation was simply turning `array[][]` into `array`, but I don't see the diff nodes affected here.

Reply:
Changing `/** @var $use Use_|GroupUse */` to `/** @param Use_|GroupUse $use */`

Reply:
Seems related to https://github.com/Roave/BetterReflection/pull/394/files#r164988430 - type is being inferred over implementation of `array_map()`. LGTM.

Reply:
Not totally related. Psalm has a `MissingDocblockType` that triggers for `@var $var Type` annotations. When I had a look at this, it made sense to not only fix the order by change it to a `@param`.

Psalm doesn't project types into closures, so it's not able to infer this, but it does check the boundaries – e.g. it sees the error in

```php
class A {
  public function foo() : void {}
}
class B {
  public function foo() : void {}
}

$as = [new A(), new B()];

$classes = array_map(
  function(B $a) : string {
    $a->foo();
    return "hello";
  },
  $as
);
```

(I don't have link generation set up, but you can always paste into [getpsalm.org](https://getpsalm.org/) to verify)

Reply:
👍